project/kg-covid-19/index.md

```diff
@@ -11,9 +11,40 @@ twitter:
 
 **KG-COVID-19: Knowledge Graph for COVID-19 Response**
 
-The goal of the KG-COVID-19 project is to use Machine Learning (ML) to make useful COVID-related predictions. Rich biological knowledge exists about SARS-CoV-2 and related viruses (SARS-CoV, MERS-CoV), for example genomics, epidemiology, susceptibility, clinical/symptoms, viral lifecycle, pathways, and inhibitors.
-However, much of this information is either locked in textual form in the literature, or in individual siloed databases, making integrative prediction difficult and costly.
-In this project, we will 1) produce a COVID-19 knowledge graph (KG) to support both DOE and international COVID-19 efforts; 2) incorporate data from COVID-19 biomedical literature in the form of natural language processing (NLP)-derived terms and deep learning embeddings; 3) apply machine learning (ML) techniques to discover new links
-(for example, identifying existing drugs that could be repurposed to treat COVID-19) and generate actionable knowledge.
+The goal of the KG-COVID-19 project is to use Machine Learning (ML) to generate useful
+COVID-related knowledge. Rich biological knowledge exists about SARS-CoV-2 and related 
+viruses (SARS-CoV, MERS-CoV), for example genomics, epidemiology, susceptibility, 
+clinical/symptoms, viral lifecycle, pathways, and inhibitors. However, much of this 
+information is either locked in textual form in the literature, or in individual 
+siloed databases, making integrative prediction difficult and costly. 
 
-![img](kg-covid-pipeline.jpg)
+To address this challenge, we have produced we have produced a COVID-19 knowledge graph
+(KG) to support both DOE and international COVID-19 efforts:
+https://github.com/Knowledge-Graph-Hub/kg-covid-19/wiki
+
+This knowledge graph incorporates up-to-date data extracted from biomedical databases 
+and literature, including drug, protein-protein interactions, SARS-CoV-2 gene 
+annotations, concept and publication data from the CORD19 data set. We also ingest
+the Human Phenotype Ontology, the Gene Ontology, and MONDO for phenotype, gene 
```

Comment:
it's now "Mondo" (not MONDO) (i know)
links to these ontologies would be a bonus